iBridges.py

```diff
@@ -233,6 +236,8 @@ def init_logger():
     def new_factory(*args, **kwargs) -> logging.LogRecord:
         """Custom record factory"""
         record = old_factory(*args, **kwargs)
+        # Limit the size of the log message to something sane.
+        record.msg = record.msg[:1024]
```

Comment:
Should this be a define at the top of the file or can this constant stay hidden in the code?

Reply:
Hmm, the number of characters is arbitrary and meant only to prevent MBs or more of file contents to enter the log file during debug-level transfers.  It certainly can be a constant to make it clearer.  Also, this code is destined to be in the `utils` module shortly in any case.

Reply:
Updated.